common/sgx/tcbinfo.c

```diff
@@ -281,24 +281,24 @@ static oe_tcb_level_status_t _parse_tcb_status(
 }
 
 oe_sgx_tcb_status_t oe_tcb_level_status_to_sgx_tcb_status(
-    const oe_tcb_level_status_t* tcb_level_status)
+    oe_tcb_level_status_t tcb_level_status)
```

Comment:
Is there specific reason to pass by value instead of constant reference?

Reply:
oe_tcb_level_status_t is a 4-byte union. Passing by pointer or reference does not save anything.